### PR TITLE
add config options for log colors

### PR DESCRIPTION
**Pull request by ghedamat**
as discussed in https://github.com/ember-cli/ember-cli-deploy/issues/358

coordinated with https://github.com/ember-cli-deploy/ember-cli-deploy-plugin/pull/8

### CODE REVIEW

File: lib/tasks/pipeline.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ module.exports = Task.extend({
     }
 
     this._pipeline = this.pipeline || new Pipeline(this.hooks, {
-      ui: this.ui
+      ui: this.ui,
+      config: this.config
```
Comment (ghedamat) [inline lib/tasks/pipeline.js]: shall we pass just a subset of `config` instead?

File: lib/models/pipeline.js [hunk 2]
```diff
@@ -14,6 +12,12 @@ var red   = chalk.red;
 function Pipeline(hookNames, options) {
   hookNames = hookNames || [];
   options = options || {};
+  var config = _.merge({
+    log: {
+      info: 'blue',
+      error: 'red'
+    }
+  }, options.config);
```
Comment (ghedamat) [inline lib/models/pipeline.js]: feedback on the structure for the options is welcome

**Comment by lukemelia** [comment]
- [ ] Need docs

**Comment by ghedamat** [comment]
@lukemelia correct, but that we need to do on `gh-pages` branch

**Pull request merged**